Adyen.podspec

```diff
@@ -59,14 +59,10 @@ Pod::Spec.new do |s|
     }
   end
 
-  # s.subspec 'AdyenCardScanner' do |plugin|
-  #   plugin.source_files = 'AdyenCardScanner/**/*.swift'
-  #   plugin.framework_name = 'AdyenCardScanner'
-  #   plugin.pod_target_xcconfig = { 
-  #     'PRODUCT_MODULE_NAME' => 'AdyenCardScanner',
-  #     'DEFINES_MODULE' => 'YES'
-  #   }  
-  # end
+  s.subspec 'CardScanner' do |plugin|
+    plugin.dependency 'Adyen/Card'
```

Comment:
Wouldn't this create cyclic dependency? Card module will be importing this module right?

Reply:
This is just a safety check but I see your point. Card scanning doesn't make sense without the Card, right? On the other hand right now Card is one of default pods.
Should I remove it?

Reply:
Yeah, I think so.